ui/src/store/resource.ts

```diff
@@ -77,6 +77,13 @@ export const Resource = types
     },
     get installCommand() {
       return `kubectl apply -f ${self.displayVersion.rawURL}`;
+    },
+    get tknInstallCommand() {
+      const versionFlag =
+        self.latestVersion.version !== self.displayVersion.version
+          ? ` --version ${self.displayVersion.version}`
+          : ``;
+      return `tkn hub install ${self.kind.name.toLowerCase()} ${self.name}${versionFlag}`;
```

Comment:
Can you please also add command if the task is from other catalog i.e can you please add command which supports multi catalog support as well